### ISSUE TITLE
Cliffy does not work (or not as desired) with Windows Powershell or commandline

### ISSUE THREAD

**Issue by NewtTheWolf** [c4spar/deno-cliffy#214]
I have written a small cmd program with an input and a checkbox

Unfortunately the display is very misleading and the checkbox doesn't work, so you can't type anything.

code:

```ts
const result = await cliffyPrompt([{
    name: "option",
    message: "What do you want to Create?",
    type: Checkbox,
    options: ["router", "migration"],
},
{
    name: "file",
    message: "What do you want to call the file?",
    type: Input,
}]);
```

Result:
![screenshot](https://user-images.githubusercontent.com/59208097/121935775-201cd800-cd49-11eb-8caf-bf2aae224a78.png)

**Comment by c4spar** [comment]
Hi @NewtTheWolf, this is a known deno bug on windows. `Deno.stdout.write` does not handle unicode characters corrcetly.
`console.log` would work but it renders also a new line. i will check if i can workaround this.

**Comment by c4spar** [comment]
i made some changes, can you check if this is working for you?
```ts
import {
  Checkbox,
  Input,
  prompt
} from "https://raw.githubusercontent.com/c4spar/deno-cliffy/prompt/windows-unicodes/prompt/mod.ts";

const result = await prompt([{
  name: "option",
  message: "What do you want to Create?",
  type: Checkbox,
  options: ["router", "migration"],
}, {
  name: "file",
  message: "What do you want to call the file?",
  type: Input,
}]);
```

**Comment by NewtTheWolf** [comment]
So

The display looks much better now, but I can only switch the options with the letter `D` (Swiss keyboard layout) and look for nothing

If I press `enter` and then give the name the options array is empty (example in the picture)
![Gif](https://owo.uwu.solutions/GAPYBME4_)
![grafik](https://user-images.githubusercontent.com/59208097/122107505-6e9aa700-ce1b-11eb-9dce-6f8811be70bc.png)

**Comment by c4spar** [comment]
Arrow keys doesnt work on windows right now. this is also a known deno issue and there is no workaround for this aktuelly. on windows you have to use`u` and `d` for `up` and `down`.
The options should be selectable with the `space` key.

**Comment by NewtTheWolf** [comment]
Then the snippet works perfectly!

Should I continue to use the snippet or is there still an update coming?

**Comment by c4spar** [comment]
I will merge it into the main branch now and i will release a new version later.

**Comment by c4spar** [comment]
is merged now. thx for double checking!

**Comment by c4spar** [comment]
@NewtTheWolf the fix is now available in v0.18.2.
